### ISSUE TITLE
Docker compose build error:  ./gradlew: not found

### ISSUE THREAD

**Issue by mahgo** [Netflix/conductor#3393]
**Describe the bug**
When building via `docker-compose build` from a fresh clone on the `main` branch, I get the following error:
```
 => ERROR [conductor:server builder 4/4] RUN ./gradlew build -x test --stacktrace                                  0.4s
------
 > [conductor:server builder 4/4] RUN ./gradlew build -x test --stacktrace:
#0 0.357 /bin/sh: 1: ./gradlew: not found
------
failed to solve: executor failed running [/bin/sh -c ./gradlew build -x test --stacktrace]: exit code: 127
```

I am attempting to build this on Windows 11 with WSL2 and Docker Desktop.

**Details**
Conductor version: latest on main branch

**To Reproduce**
Follow steps as per: https://conductor.netflix.com/gettingstarted/docker.html

**Expected behavior**
Docker compose command succeeds.

**Comment by ikshvakoo** [comment]
mahgo - you will need go to the root directory of conductor on your machine and run "gradle wrapper" command. This will install the wrapper and you'll get the gradlew file at the root. Once you've completed running gradle wrapper command successfully, run "docker-compose build"

**Comment by ikshvakoo** [comment]
That's where I'm stuck as well. See the other bug I opened yesterday for this.. No response yet.

**Comment by manan164** [comment]
Hi @mahgo ,
Please follow the steps,
1. Delete your current conductor folder
2. Run **git config --global core.autocrlf false**; 
3. Run **git config --global core.eol lf**; (Small L and f)
4. Clone conductor repo again
5. Check if end of lines shows as CRLF via NotePad++ by opening a DockerFile, e.g. => conductor/docker/server/DockerFile
6. Run docker-compose up on conductor/docker

**Comment by mahgo** [comment]
Hi @manan164,

Thanks for that, that fixed the issue.

**Comment by onderhamamcioglu** [comment]
if you have git bash installed try running `unix2dos Dockerfile`
that fixed it for me on conductor 3.13.2
